tests/tests.rs

```diff
@@ -29,7 +29,7 @@ fn test_hash_algorithm_compat() {
 }
 
 fn test_hash(data: &[u8], eq_to: u64) {
-    let mut hasher_7 = ahash::AHasher::new_with_keys(1, 2);
+    let mut hasher_7 = pearl::ahash::AHasher::new_with_keys(1, 2);
```

Comment:
Probably, this is the reason you exported `ahash` from the crate. I suggest to move `test_hash_algorithm_compat` into `bloom.rs`, because the compatibility is required only for bloom filter